app/views/admin/content_assets/edit.html.erb

```diff
@@ -1,11 +1,14 @@
+<%= link_to 'Back', admin_content_assets_path, class: 'govuk-back-link' %>
+
 <h1 class="govuk-heading-l">Edit Asset</h1>
 
 <% if @content_asset.asset_file.attached? %>
 
 <div id='content-asset-wrapper'>
     <%= image_tag(url_for(@content_asset.asset_file), :class=>'govuk-!-width-full') %>
     <div id='asset-details'>
-    <h3>Asset Details</h3>
+    <br/>
+    <h3 class="govuk-heading-s">Asset Details</h3>
```

Comment:
Rather than using a `br`, use the [appropriate GOV.UK styling class](govuk-!-padding-bottom-3) to get the desired spacing. For example `govuk-!-margin-top-3`